### PR TITLE
/var options and /var/log /var/log/audit and /usr

### PR DESCRIPTION
**Pull request by monsieuremre**
nosuid recommended. flatpaks are by default installed on /var, so noexec would break them. adjusted accordingly

### CODE REVIEW

**Comment by monsieuremre** [comment]
added /var /var/log /var/log/audit and /usr. everything commented out. for testing only.

File: usr/bin/remount-secure [hunk 1]
```diff
@@ -188,6 +187,44 @@ _var_tmp() {
...
+#   mount_folder="$NEWROOT/var/log/audit"
+#   intended_mount_options="nosuid,nodev,noexec"
+#   remount_secure "$@"
+#}
+
+# Not tested
+#
+# Under /etc/apt/apt.conf.d/
+# If a config file is created with the following content:
+# Dpkg::Pre-Invoke {"mount -o remount,nodev,rw /usr";};
```
Comment (adrelanos) [inline usr/bin/remount-secure:216]: This won't work because APT needs write access to update the package lists.

**Comment by adrelanos** [comment]
Anything left do here?

Merge conflict.

And looks incomplete either way.

**Comment by monsieuremre** [comment]
You can close the request. Most everything seems to be incorporated in the new script anyway. This was more of a scetch which is no longer necessary.

**Pull request closed without merging**